### PR TITLE
improve orchestration retry cmd

### PR DESCRIPTION
**Pull request by lauraanddolay**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

File: components/kyma-environment-broker/internal/orchestration/handlers/kyma_retry.go [hunk 1]
```diff
@@ -60,17 +60,19 @@ func (r *kymaRetryer) orchestrationRetry(o *internal.Orchestration, opsByOrch []
...
+		resp.RetryShoots = append(resp.RetryShoots, op.Operation.InstanceDetails.ShootName)
 	}
 	resp.Msg = "retry operations are queued for processing"
 
-	err = r.OperationsStateUpdate(ops, immediate)
-	if err != nil {
-		return resp, err
+	for _, op := range ops {
+		fmt.Println("kyma_retry.go o.Parameters.RetryOperation.Immediate=", o.Parameters.RetryOperation.Immediate)
+		fmt.Println("kyma_retry.go o.Parameters.RetryOperation.RetryOperations=", o.Parameters.RetryOperation.RetryOperations)
```
[the remark this answers is not in the document]
Comment (lauraanddolay) [reply components/kyma-environment-broker/internal/orchestration/handlers/kyma_retry.go]: Deleted the `fmt.Println()`.

File: components/kyma-environment-broker/internal/orchestration/handlers/kyma_retry.go [hunk 2]
```diff
@@ -60,17 +60,19 @@ func (r *kymaRetryer) orchestrationRetry(o *internal.Orchestration, opsByOrch []
 	}
 
 	for _, op := range ops {
-		resp.RetryOperations = append(resp.RetryOperations, op.Operation.ID)
+		resp.RetryShoots = append(resp.RetryShoots, op.Operation.InstanceDetails.ShootName)
 	}
 	resp.Msg = "retry operations are queued for processing"
 
-	err = r.OperationsStateUpdate(ops, immediate)
```
[the remark this answers is not in the document]
Comment (lauraanddolay) [reply components/kyma-environment-broker/internal/orchestration/handlers/kyma_retry.go:67]: Hi @ebensom  now i removed the unused `OperationsStateUpdate` function.

File: components/kyma-environment-broker/common/orchestration/dto.go [hunk 3]
```diff
@@ -15,7 +15,12 @@ type Parameters struct {
 	// upgrade kyma specific parameters
 	Kyma *KymaParameters `json:"kyma,omitempty"`
 	//customer notification status
-	NotificationState notificationStateType `json:"notificationstate,omitempty"`
+	NotificationState notificationStateType    `json:"notificationstate,omitempty"`
+	RetryOperation    RetryOperationParameters `json:"retryoperation,omitempty"`
+}
+type RetryOperationParameters struct {
+	RetryOperations []string `json:"retryoperations,omitempty"`
+	Immediate       string   `json:"immediate,omitempty"`
```
[the remark this answers is not in the document]
Comment (lauraanddolay) [reply components/kyma-environment-broker/common/orchestration/dto.go]: Latest commit changed to `bool` type.

**Review by ebensom — APPROVED** [review]

**Review by ebensom — APPROVED** [review]

**Pull request merged**